src/figuresClasses.ts

```diff
@@ -1,19 +1,110 @@
 export interface Figure {
-
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: 'triangle';
 
-}
+  color: 'red' | 'green' | 'blue';
+
+  sideA: number;
+
+  sideB: number;
+
+  sideC: number;
 
-export class Circle {
+  constructor(
```

Comment:
you can access Figure types with this syntax: `Figure['typeName']` - you won't have to retype everything again